src/components/ui.js

```diff
@@ -0,0 +1,38 @@
+
+import Weather from './request';
+
+const searchbtn = document.querySelector('#change-btn');
+
+
+const weather = new Weather(document.querySelector('#city').value, document.querySelector('#country').value);
+
+const populateUi = () => {
+  const getReport = () => {
+    weather.getWeather()
+      .then(result => {
+        document.querySelector('#location').textContent = `${result[1].country}`;
+        document.querySelector('#description').textContent = `The weather will be ${result[0][0].description}`;
+        document.querySelector('#temp').textContent = `Temperature is ${result[2].temp}`;
+      })
+      .catch(() => {
+        const errorMessage = document.querySelector('#error');
+        errorMessage.setAttribute('class', 'alert alert-warning');
+        errorMessage.setAttribute('role', 'alert');
+        errorMessage.textContent = 'PLEASE SEARCH FOR A VALID CITY AND COUNTRY';
+      });
+  };
+
+
+  document.addEventListener('DOMContentLoaded', getReport);
+  searchbtn.addEventListener('click', () => {
+    const cityInput = document.querySelector('#city').value;
+    const countryInput = document.querySelector('#country').value;
+
+    weather.changeLocation(cityInput, countryInput);
+    getReport();
```

Comment:
## Thumbs up 👍 & Suggestion ❗ 
✅ This is the first project I have seen with the query option of a country as well though I realised you can query the city alone without the country. 

##### Suggestion
- [ ] Does it fill redundant having the country field as part of the form, though the `weathermapapi` will automatically detect that for you? I would suggest we remove it and instead leave the form with only the city field to query the API. 
- [ ] I would also suggest we get to have the name of the city displaying in the result page of the UI after the query is successful instead of only having the country's code showing.  A user might want to query different types of cities in the same country and they would know which is which since the country code and different results of it will be showing. What's your take on this?